packages/ui-flex/src/Flex/__examples__/Flex.examples.tsx

```diff
@@ -61,7 +61,8 @@ const grow = [
 ]
 
 export default {
-  maxExamplesPerPage: 50,
+  maxExamplesPerPage: 20,
+  maxExamples: 100,
```

Comment:
While not an elegant solution, it will do for now. We need a better example filtering here in the future; a few hundred examples of `Flex` must be enough for a thorough visual test.

Reply:
You are right, fixed